src/daemon/demo.sh

```diff
@@ -302,30 +302,23 @@ function bootstrap_mgr {
 # SREE #
 ########
 function bootstrap_sree {
-  if [ ! -f sree.tar.gz ]; then
-    if [ -z "$SREE_VERSION" ]; then
-      sree_latest=$(curl -s 'https://api.github.com/repos/leseb/Sree/releases/latest' | grep tarball_url | cut -d '"' -f 4)
-      curl -L "$sree_latest" -o sree.tar.gz
-    else
-      curl -L https://github.com/leseb/Sree/archive/"$SREE_VERSION".tar.gz -o sree.tar.gz
-    fi
-    mkdir sree && tar xzvf sree.tar.gz -C sree --strip-components 1
```

Comment:
Why do you remove the tar xvzf ?
I don't get how you extract the tarball then.

Reply:
The tarball gets extracted by Docker during build time .

Reply:
New approach please have a look.

Reply:
New approach please have a look.